### DISCUSSION TITLE
How do I use Java native functions? for example math

### DISCUSSION THREAD

**Discussion by maxima120** [sebastienros/jint#1524] [Q&A]
How do I use native java math functions? is for example object `Math` available? what if I need special functions? for example mathjs.org library?

**Comment by Genteure** [comment]
"java math functions" are not available, but [javascript `Math`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math) are.
[Java](https://en.wikipedia.org/wiki/Java_(programming_language)) and [JavaScript](https://en.wikipedia.org/wiki/JavaScript) are completely different thing, they are not related in the slightest (except their name both contains `java`)

mathjs.org should work just fine, did you encounter any problem?
